src/ode_premade_problems.jl

```diff
@@ -294,27 +294,42 @@ Usually solved from 0 to 20.
 prob_ode_rigidbody = ODEProblem(rigid,[1.0,0.0,0.9],(0.0,20.0))
 
 # Pleiades Problem
-
-pleides = (t,u,du) -> begin
-  x = view(u,1:7)   # x
-  y = view(u,8:14)  # y
-  v = view(u,15:21) # x′
-  w = view(u,22:28) # y′
-  du[1:7] .= v
-  du[8:14].= w
-  for i in 14:21
-    du[i] = zero(u)
+numberofstars = 7
```

Comment:
No, this is the problem library version. It should be set and simple. And the Pleiades problem is a very specific problem which is not the N-body problem. Develop the general version as a package function in DiffEqPhysics.jl